tests/integration_tests/test_skorch.py

```diff
@@ -0,0 +1,47 @@
+from typing import Any
+
+import skorch
+from skorch.helper import predefined_split
+import torch
+from torch import nn
+
+import optuna
+from optuna.integration import SkorchPruningCallback
+from optuna.testing.integration import DeterministicPruner
+
+
+class ClassifierModule(nn.Module):
+    def __init__(self) -> None:
+        super(ClassifierModule, self).__init__()
+        self.dense0 = nn.Linear(4, 8)
+
+    def forward(self, X: torch.Tensor, **kwargs: Any) -> torch.Tensor:
+        return self.dense0(X)
+
+
+def test_skorch_pruning_callback() -> None:
+
+    X, y = torch.zeros(3, 4), torch.zeros(3, dtype=torch.long)
+    valid_ds = torch.utils.data.TensorDataset(torch.zeros(3, 4), torch.zeros(3, dtype=torch.long))
+
+    def objective(trial: optuna.trial.Trial) -> float:
+
+        net = skorch.NeuralNetClassifier(
+            ClassifierModule,
+            max_epochs=10,
+            lr=0.02,
+            callbacks=[SkorchPruningCallback(trial, "valid_acc")],
+            train_split=predefined_split(valid_ds),
```

Comment:
In my environment, an error was occurred with `skorch==0.9.0`.  It works with `skorch==0.8.0`, and the new version may break compatibility.

Reply:
Fixed this in https://github.com/optuna/optuna/pull/1668/commits/ce004db0d68ff0190ae6e3f9d59b9e56e3bc32df.